src/utilities/tokens.ts

```diff
@@ -0,0 +1,53 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import tokens from '@frontify/fondue-tokens';
+
+export type PalleteName = 'base' | 'box' | 'focus' | 'line' | 'overlay' | 'shadow' | 'button';
+export type TokenPallete = {
+    name: PalleteName;
+    attributes: {
+        'tailwind-name': string;
+    };
+};
+
+export type TokenPalletes = Record<PalleteName | string, TokenPallete[]>;
+
+export const TOKEN_STATES = ['hover', 'pressed', 'inverse', 'inverse-hover', 'inverse-pressed'];
+export const TOKEN_STATES_REGEX = new RegExp(`(${TOKEN_STATES.join('|')})$`, 'i');
```

Comment:
That looks overkill, what is the intention?

Reply:
The intention here is to get only the "base" token name from the list of tokens, specially when using box tokens. That base name will be used as a prefix for the states one. The regex will be used to filter those and retrieve a list with only the base ones. 
Example:
```
// The first three ones should be applied to bg
box-neutral - added to the list
box-neutral-hover - ignored
box-neutral-pressed - ignored
// inverse ones should be applied to color
box-neutral-inverse - ignored
box-neutral-inverse-hover - ignored
box-neutral-inverse-pressed - ignored
```